backend/src/controllers/otp.ts

```diff
@@ -1,5 +1,5 @@
 import sendEmail from "../utils/email";
-import OTP from "../models/otp"
+import OTP from "../models/OTP";
```

Comment:
Always respect consistency in a project, all file names are small so please take care of this